libass/ass.h

```diff
@@ -388,6 +388,18 @@ void ass_renderer_done(ASS_Renderer *priv);
  */
 void ass_set_frame_size(ASS_Renderer *priv, int w, int h);
 
+/**
+ * \brief Set the number of threads to use during rendering. Default is 0,
+ * meaning the number of available logical cores if known, otherwise 1.
+ * \param priv renderer handle
+ * \param threads number of threads
+ *
+ * NOTE: this can only be configured before the first frame is rendered.
+ * Calling this function requires the log callback of the associated ASS_Library
+ * to be thread-safe.
+ */
+void ass_set_threads(ASS_Renderer *priv, unsigned threads);
```

Comment:
Do you think it’s a good idea to return a `bool` here, so API users could detect and adjust to builds with and without threading support?

Reply:
I can't imagine anything a sane threading-aware caller would want to do differently in the non-threaded case.